revapi/src/main/java/org/revapi/DifferenceTransform.java

```diff
@@ -55,6 +58,29 @@ public interface DifferenceTransform<E extends Element<?>> extends AutoCloseable
      */
     Pattern[] getDifferenceCodePatterns();
```

Comment:
Let's mark this as deprecated and provide a default impl so that new implementations of this interface can start using just the predicates variant instead of also having to implement this method.

Reply:
That sounds good with me. Will make the change to this being `default` returning an empty `Pattern[]`